specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-11-01/AssetsAndAssetFilters.json

```diff
@@ -0,0 +1,2032 @@
+        }
+      },
+      "type": "object",
+      "description": "Represents a text track in an asset. A text track is usually used for sparse data related to the audio or video tracks."
+    },
+    "AssetTrackProperties": {
+      "properties": {
+        "track": {
+          "$ref": "#/definitions/TrackBase",
+          "description": "Detailed information about a track in the asset."
+        },
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Failed",
+            "InProgress",
+            "Succeeded"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "values": [
+              {
+                "value": "Failed",
+                "description": "Provisioning state failed."
+              },
+              {
+                "value": "InProgress",
+                "description": "Provisioning state in progress."
+              },
+              {
+                "value": "Succeeded",
+                "description": "Provisioning state succeeded."
+              }
+            ],
+            "modelAsString": true
+          },
+          "description": "Provisioning state of the asset track.",
+          "readOnly": true,
+          "x-nullable": false
+        }
+      },
+      "type": "object",
+      "description": "Properties of a video, audio or text track in the asset."
+    },
+    "AssetTrack": {
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AssetTrackProperties",
+          "description": "The resource properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "description": "An Asset Track resource."
+    },
+    "AssetTrackOperationStatus": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Operation identifier."
+        },
+        "id": {
+          "type": "string",
+          "description": "Operation resource ID."
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Operation start time."
+        },
+        "endTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Operation end time."
+        },
+        "status": {
+          "type": "string",
+          "description": "Operation status."
+        },
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorDetail",
+          "x-nullable": true
+        }
+      },
+      "type": "object",
+      "required": [
+        "name",
+        "status"
+      ],
+      "description": "Status of asset track operation."
+    },
+    "AssetTrackOperationResult": {
```

Comment:
why is this a different model than "AssetTrack"?

Reply:
changed to AssetTrack